packages/manager/modules/pci/src/projects/project/storages/databases/components/editable-text/editable-text.controller.js

```diff
@@ -0,0 +1,56 @@
+import uniqueId from 'lodash/uniqueId';
+
+export default class {
```

Comment:
Can we name the controller

Reply:
Done here: https://github.com/ovh/manager/pull/6399/commits/8bc3b319b24e5d02147807e0ad3e47c257603cd9